File: A01/b.-estructurado-seleccion/parte2/2/c++/main.cpp
#include <iostream>
using namespace std;

int main(){
	int calif;
	cout<<"Ingresa una calificacion: ";
	cin>>calif;
	cout<<"Calificacion ingresada "<<calif<<endl;

	if(calif<60){
		cout<<"Eres un desastre!";
	}else if(calif<70){
		cout<<"Apenas la libraste";
	}else if(calif<80){
		cout<<"Vas por buen camino";
	}else if(calif<90){
		cout<<"Me impresionas!";
	}else if(calif<=99){
		cout<<"Casi alcanzas la divinidad";
	}else if(calif==100){
		cout<<"Eres un Dios!";
	}else cout<<"Numero fuera de rango";
	cout<<endl;

	return 0;
}

File: A01/b.-estructurado-seleccion/parte2/2/python/main.py
calif = int(input("Ingresa una calificación: "))
print ("Calificación ingresada {}".format(calif))

if calif < 60:
	print("¡Eres un desastre!")
elif calif < 70:
	print("Apenas la libraste")
elif calif < 80:
	print("Van por buen camino")
elif calif < 90:
	print("¡Me impresionas!")
elif calif <=99:
	print("Casi alcanzas la divinidad")
elif calif == 100:
	print("¡Eres un Dios!")
else:
	print("Número fuera de rango")

